doc/product-reference.md

```diff
@@ -144,6 +145,17 @@ Specifies product method
 | Name | `friendlyName` | string | Method name | Yes |
 | Description | `description` | string | Method description message | No |
 
+#### PolicyConfigSpec
+
+Specifies product policy config object
+
+| **Field** | **json field**| **Type** | **Info** | **Required** |
+| --- | --- | --- | --- | --- |
+| Name | `name` | string | Policy name | Yes |
+| Version | `version` | string | Policy version | Yes |
+| Enabled | `enabled` | boolean | Policy enabling switch | Yes |
+| Configuration | `configuration` | object | Policy configuration object | Yes. Minimum required is the empty object `{}` |
```

Comment:
Is there some doc we can link to about the configuration spec format or something?

Reply:
AFAIK, there is not.